client/src/index.js

```diff
@@ -3,5 +3,11 @@ import ReactDOM from 'react-dom';
 
 import './index.css';
 import App from './App';
+import {BrowserRouter as Router} from 'react-router-dom';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+<Router>
+<App />
```

Comment:
make sure to indent properly
```suggestion
    <App />
```